lib/person.rb

```diff
@@ -0,0 +1,29 @@
+require 'date'
+require 'pry-byebug'
+require './lib/library.rb'
+
+class Person
+
+  attr_accessor :library
+
+  def initialize
+    @library = Library.new
+  end
+
+
+  def check_books
+    @library.books_instock
+  end
+
+  def checkout(title, author)
+    @library.book_checkout(title, author)
+  end
+
+  def check_date(title, author)    #Individual should be able to return date
+    check_date =@library.search_book(title, author)
+    if check_date[:available] == false
```

Comment:
In ruby, you don't have to explicitly check if a value is `true` or `false` using `==`
and so

```ruby
if value == true
  # do something
end
```
 is the same as
```ruby
if value
  # do something
end
```
if the value evaluates to `true` then the code block is executed otherwise, it's not.